Withdrawn/CustomDialogOnClose/explainer.md

```diff
@@ -4,8 +4,11 @@
 
 -   Austin Orion
 
+## Withdrawn
+This explainer has been withdrawn. The customer need has been met by other means, and this design conflicted with the philosophy of keeping tab close as fast as possible. In order to accomplish actions on tab close, developers should try writing to IndexedDB on the  `visibilitychange` event, and then trigger a Service Worker to read from the DB and do any other necessary work.
```

Comment:
> The customer need has been met by other means

As we still see an issue with using other means here, I believe we should not say this and rather mention we plan to work on it to further clarify the edge cases the suggested approach doesn't fully cover at the moment.

Reply:
How does it look now?